setup.py

```diff
@@ -14,8 +19,9 @@
     author_email='info@scrapinghub.com',
     url='http://github.com/scrapinghub/python-scrapinghub',
     platforms = ['Any'],
-    py_modules = ['scrapinghub'],
-    install_requires = ['requests'],
+    py_modules = ['scrapinghub', 'hubstorage'],
```

Comment:
> I guess we're going to have a single version for the merged repository.
@vshlapakov sure. However, I'd like to postpone release a bit - e.g. to try version from master after merge in some internal services before publishing it to PyPI. So let's setup it properly but not change the version yet.

Regards approach - you can use approach from scrapy, just don't forget to include VERSION file in package data (or MANIFEST.in). Or you can stick with current setup with bumpversion - add __version__ variable to scrapinghub/__init__.py, mention this file and setup.py in bumpversion config.

We are going to bump python-scrapinghub version 1.8.0 -> 1.9.0 - right?